src/frontend/main.go

```diff
@@ -120,12 +120,12 @@ func main() {
 	go initTelemetry(log)
 
 	// TODO: register views when OpenTelemetry Views API is available
-	meter := controller.Provider().Meter("hipstershop/frontend")
+	meter := controller.Provider().Meter("hipstershop")
```

Comment:
Is this change related to product catalog service?

Reply:
removed this change